lib/matplotlib/_animation.py

```diff
@@ -0,0 +1,210 @@
+# Javascript template for HTMLWriter
```

Comment:
Perhaps `_animation_data.py`? (like `_color_data.py`)

Reply:
I like it.